README.md

```diff
@@ -73,6 +73,25 @@ Whenever the same functionality can be achieved in multiple offerings but in a d
 
 For links within this documentation, please end the link with `/index.md`. This way all links will be checked, and you’ll get an error if a link is broken at any point.
 
+### Concepts
+
+There are several key concepts at Snowplow: events (self-describing, structured), entities, schemas. We must ensure that we use and explain them consistently.
```

Comment:
Nit: "in Snowplow", "at Snowplow" feels directed to staff (in Snowplow the technology vs at Snowplow the company).